src/sentry/utils/snuba.py

```diff
@@ -371,7 +371,7 @@ def get_arrayjoin(column):
         return match.groups()[0]
 
 
-def transform_aliases_and_query(*args, **kwargs):
+def transform_aliases_and_query(skip_conditions=False, *args, **kwargs):
```

Comment:
Having a keyword argument before `*args` is going to make this function hard to call with positional arguments and `skip_conditions` as a keyword argument.

```python
>>> def tester(skip=False, *args, **kwargs):
...   print('%s %s %s', skip, args, kwargs)

>>> tester(1, 2, 3, skip=True, a='b')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: tester() got multiple values for keyword argument 'skip'
```

Reply:
Ahh, thanks! I removed `*args` since we don't actually pass them into this function